src/WebManager/Controllers/ApiController.cs

```diff
@@ -78,6 +80,33 @@ public RegistryCredential GetDockerCredential()
             };
         }
 
+    
+
+        [HttpPut]
+        public async Task<IActionResult> Manifest(string repo, string tag,  [FromBody] string manifest)
+        {
+
+            string json = manifest;
```

Comment:
Sorry, that was from when I still used an auxiliary class and I had to build the instance, json was the instance